lib/bokken/events.ex

```diff
@@ -33,6 +34,17 @@ defmodule Bokken.Events do
     |> Map.fetch!(:teams)
   end
 
+  def list_teams(%{"event_id" => event_id}) do
```

Comment:
I think this function gives us the team of an event. Am I right? If so, we don't really need this, we just do `Repo.get_event!(event_id, [:team]).team`.

Reply:
If this function isn't needed I'll remove it. Is it ok with you?